lib/services/ios-project-service.ts

```diff
@@ -311,12 +328,85 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 				args.push(`DEVELOPMENT_TEAM=${buildConfig.teamIdentifier}`);
 			}
 
+			// this.$logger.out("xcodebuild...");
 			this.$childProcess.spawnFromEvent("xcodebuild", args, "exit", { cwd: this.$options, stdio: 'inherit' }).wait();
+			// this.$logger.out("xcodebuild build succeded.");
+
 			this.createIpa(projectRoot).wait();
 
 		}).future<void>()();
 	}
 
+	private setupManualSigning(projectRoot: string, buildConfig?: IiOSBuildConfig): IFuture<void> {
+		return (() => {
+			if (this.$options.provision) {
+				const pbxprojPath = path.join(projectRoot, this.$projectData.projectName + ".xcodeproj", "project.pbxproj");
+				const xcode = Xcode.open(pbxprojPath);
+				const signing = xcode.getSigning(this.$projectData.projectName);
+
+				let shouldUpdateXcode = false;
+				if (signing && signing.style === "Manual") {
+					for(let config in signing.configurations) {
+						let options = signing.configurations[config];
+						if (options.name !== this.$options.provision && options.name !== this.$options.provision) {
```

Comment:
both statements check the same conditions, maybe we really want to be sure that `options.name !== this.$options.provision` 😄